book/src/07_parallel_join.md

```diff
@@ -44,6 +44,8 @@ method call can be exchanged for `par_join`:
     }
 ```
 
-The `par_join` method produces a type implementing `rayon`s `ParallelIterator`
+The `par_join` method produces a type implementing [`rayon`\`s `ParallelIterator`][ra]
```

Comment:
Wouldn't you use `'` instead of ` here?

Reply:
ah ok, I am not a natural speaker too, so sorry for the misspelling